packager/VisibilityChecker.cc

```diff
@@ -503,6 +555,8 @@ class VisibilityCheckerPass final {
             return;
         }
 
+        symbolsReferenced.insert(litSymbol);
```

Comment:
Given a constant like `A::B::C::D`, this will only add a reference to `::A::B::C::D`, and not any of the symbols in the scope. Is that what we want? I haven't looked, but I wanted to make sure to ask in case you already thought about it.